Userland/Libraries/LibCore/FileSystem.cpp

```diff
@@ -267,11 +277,11 @@ ErrorOr<void, CopyError> copy_directory(DeprecatedString const& dst_path, Deprec
     auto my_umask = umask(0);
     umask(my_umask);
 
-    if (chmod(dst_path.characters(), src_stat.st_mode & ~my_umask) < 0)
+    if (System::chmod(dst_path, src_stat.st_mode & ~my_umask).is_error())
         return CopyError { errno, false };
```

Comment:
I'm not sure whether we can/should trust `errno` when using the `Core::System` wrappers, so we should probably store the returned `ErrorOr` and use `.error().errno()` (or however we named that function) instead.